style.css

```diff
@@ -107,3 +107,266 @@ a:link {
   align-items: flex-start;
   gap: 16px;
 }
+
+.works {
+  margin-top: 150px;
+}
+
+.project {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto;
+  margin-left: 30px;
+}
```

Comment:
- [ ] You did a great job on the project, however, the instruction is to use a grid for the main `section` that wraps your `work` section, then use flexbox to display the cards. based on your design, the section that wrapped them has the class of `works`, which should be a grid, then the different cards in class `project` should be displayed with flexbox.
- [ ] You did put in a lot o effort 👏 , but let's stick to the project requirement 
# Instruction:
![image](https://user-images.githubusercontent.com/50990925/212409587-ee179105-6d5a-482e-ad52-63ae9740fde6.png)

Reply:
The styling was corrected. Grid was styling was setup to works section and flexbox to its projects.